src/js/components/awardv2/contract/ContractContent.jsx

```diff
@@ -56,12 +57,24 @@ export default class ContractContent extends React.Component {
                     </AwardSection>
                 </AwardSection>
                 <AwardSection type="row">
-                    <AwardAmountsSection
-                        awardType={this.props.overview.category}
-                        jumpToSection={this.props.jumpToSection}
-                        awardOverview={this.props.overview}
-                        tooltipProps={defaultTooltipProps} />
-                    <ComingSoonSection title="Description" includeHeader />
+                    <AwardSection type="column">
+                        <AwardAmountsSection
+                            awardType={this.props.overview.category}
+                            jumpToSection={this.props.jumpToSection}
+                            awardOverview={this.props.overview}
+                            tooltipProps={defaultTooltipProps} />
+                    </AwardSection>
+                    <AwardSection type="column">
+                        <ComingSoonSection title="Description" includeHeader />
```

Comment:
The coming soon section already has the `<AwardSection type='column'>` in it so we should just do [this](https://github.com/fedspendingtransparency/usaspending-website/pull/1216/files?file-filters%5B%5D=.jsx&hide-deleted-files=true#diff-0563f5bfe1b06c58c0a566c0bbd040baL64)